Blog: Ways PostHog can help you build better products (#3751)
PostHog/posthog.com: contents/blog/things-you-can-do-in-posthog.md

---
date: 2022-07-13
title: 22 ways PostHog can help you build better products
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: ../images/blog/product-people.png
featuredImageType: full
author: ["andy-vandervell"]
categories: ["Guides", "Product analytics"]
---

PostHog is a growing platform.

We used to call ourselves a product analytics platform, but product analytics is just one feature [among many](/product) these days.

And that got us to wondering... just how many useful things can you do with PostHog?

The answer? A lot – and this list is just a snapshot of the posibilities.

### 1. Replace multiple services with just PostHog

Let's start with a big one.

You could use Amplitude for product analytics, Hotjar for session recording, Flagsmith for feature control, Segment as your customer data platform, Optimizely for testing, and Redshift for your data warehouse – these are sensible choices.

> Alternatively, you can just use PostHog.

Reviewer: ```suggestion
Alternatively, you may be able to just use PostHog.
```

Reviewer: since we aren't at 100% parity across all these

Author: Is there any particular one we're weak on? I'd rather take one out and keep the stronger wording if we can.

Alternatively, I might tweak how we approach this.

> Alternatively, you can just use PostHog.

Author: ```suggestion
But maybe you can replace all of them with just PostHog?
```

And if you self-host PostHog, you get a built-in data warehouse based on ClickHouse into the bargain. 

## 2. Build an AARRR pirate metrics dashboard

AARRR (Acquisition, Activation, Retention, Revenue, Referral) is a classic framework for driving product-led growth, and it's a great place to start when building your first dashboard in PostHog.

What metrics you choose will depend on this business and it's fine to have more than one metric for each category. What's vital, however, is that they're easy to understand, actionable and comparable.

Read our guide to [getting started with the AARRR framework](/tutorials/aarrr-framework) and [B2B product metrics](/blog/b2b-saas-product-metrics) for more in-depth guidance on the framework and useful metrics to measure. 

## 3. Use feature flags as kill switches

[Feature flags](/docs/user-guides/feature-flags) are often used to turn new features on under certain conditions, so that you can test things with a certain [cohort](/docs/user-guides/cohorts) or user segment. But you can also use them globally, then leverage the flag as a kill switch to turn features off in the event of an emergency. 

> Some PostHog users, such as [Phantom](/customers/phantom), aren't able to deploy new updates or features to all users instantly. In Phantom's case this is because the product is a browser extension. Using Feature Flags as kill switches in this way offers a degree of control not normally available to such products.

Reviewer: ```suggestion
Some PostHog users, such as [Phantom](/customers/phantom), didn't used to be able to deploy new updates or features to all users instantly. In Phantom's case this is because the product is a browser extension. Using Feature Flags as kill switches in this way offers a degree of control not normally available to such products.
```
